pyxrf/model/load_data_from_db.py

```diff
@@ -335,16 +388,25 @@ def map_data2D_srx(runid, fpath,
 
     else:
         # srx fly scan
+
+        print(f"****************************************")
+        print(f"         Loading SRX fly scan           ")
+        print(f"****************************************")
+        
         if save_scalar is True:
             scaler_list = ['i0', 'time', 'i0_time', 'time_diff']
             xpos_name = 'enc1'
             ypos_name = 'hf_stage_y' # 'hf_stage_x' if fast axis is vertical
+
+        num_det = 0  # Some default value (should never be used)
+
+            
         # Added by AMK to allow flying of single element on xs2
-        if 'E_tomo' in start_doc['scaninfo']['type']:
-            num_det = 1
-            ypos_name = 'e_tomo_y'
-        else:
-            num_det = 3
+        #if 'E_tomo' in start_doc['scaninfo']['type']:
+        #    num_det = 1
+        #    ypos_name = 'e_tomo_y'
+        #else:
+        #    num_det = 3
```

Comment:
Why was this block commented out?

Reply:
This condition is no longer valid. I am keeping the commented code for reference for now.